### PR TITLE
Add typings file to support typescript

### PR DESCRIPTION
**Pull request by miller45**
Typings file enables compilation and code completion support for typescript users
@ashiguruma Please have a look

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -83,3 +83,11 @@ datasets: [{
 In order to provide an alternative view for visually impaired viewers it's a good idea to [provide a patterned alternative](http://betweentwobrackets.com/data-graphics-and-colour-vision/#patternstotherescue).
 
 See the [pattern option example page](examples/optional.html) for a simple implementation of a pattern toggle switch for Chart.js.
+
+##Typescript / Angular
```
Comment (ashiguruma) [inline README.md]: @miller45 If you can just clean up the formatting of the Markdown here I should be able to get this merged. You can check it easily in the GitHub editor.

File: package.json [hunk 2]
```diff
@@ -1,9 +1,10 @@
 {
   "name": "patternomaly",
-  "version": "1.3.0",
+  "version": "1.3.1",
```
Comment (ashiguruma) [inline package.json]: @miller45 Can you also leave the version number as 1.3.0. Changing this on the branch could conflict with another feature being merged and tagged.

Comment (miller45) [reply package.json]: reverted it in new commit

File: README.md (hunk 1 above)
Comment (miller45) [reply README.md]: @ashiguruma ok I added a space and pushed resolve

**Pull request merged**